code/game/objects/machinery.dm

```diff
@@ -29,6 +29,7 @@
 	. = ..()
 	GLOB.machines += src
 	component_parts = list()
+	START_PROCESSING(SSmachines, src)
```

Comment:
Does every machine need to process on init?

Reply:
If we want a machine to consume power we do.
If a machine doesn't it will be removed on the first tick.

Reply:
Now that I think it a bit more, maybe starting either in LateInitialize or after the game mode starts might be the best here. The latter would be ideal IMO, but out of scope. Push it to late init?

Reply:
Sure